vnext/JSI.Desktop.UnitTests/JSI.Desktop.UnitTests.vcxproj

```diff
@@ -50,6 +50,8 @@
         wchar_t* str = L"hello";
       from compiling. -->
       <AdditionalOptions>%(AdditionalOptions) /Zc:strictStrings</AdditionalOptions>
```

Comment:
@hansenyy Does Facebook make these unit tests public in newer versions of React Native? If not we should try to get these exposed as part of their npm package.